packages/elements-core/src/components/TryIt/Parameters/parameter-utils.ts

```diff
@@ -36,8 +37,10 @@ export function exampleOptions(parameter: ParameterSpec) {
     : null;
 }
 
-export function parameterSupportsFileUpload(parameter: Pick<ParameterSpec, 'schema'>) {
+export function parameterSupportsFileUpload(parameter: Pick<ParameterSpec, 'schema'> | undefined) {
   return (
+    parameter &&
```

Comment:
can we  just use the `?.` for each of these after parameter?

Reply:
fixed in 934903fa